### PR TITLE
Homework 1 - 99 bottles or beer & Madlibs

### PR DESCRIPTION
**Pull request by DanPitrowiski**
- top level index.html file that will link all the homeworks
- My github name is `danpitrowiski` so I just named my file that

### CODE REVIEW

File: danpitrowiski/hw1-99-bottles-of-beer/css/style.css [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+}
+
+button:hover {
+  background-color: #4A90E2;
+  color: white;
+}
+
+/*******************************************/
+/* Bubbles
+/*******************************************/
```
Comment (Borgaard) [inline danpitrowiski/hw1-99-bottles-of-beer/css/style.css:141]: :champagne:

File: danpitrowiski/hw1-99-bottles-of-beer/app.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+let beerTotal = 10;
+let beerMax = beerTotal;
+
+
+function bottlePluralTest(number){
+	if (number === 1){
+		return "bottle";
+	}
+	else {
```
Comment (Borgaard) [inline danpitrowiski/hw1-99-bottles-of-beer/app.js:22]: _looks around_ 
`else {`? Else what?!

Who left this `else` out here all alone? Hey, Dan! Some monster dropped this`} else {` statement from its `if {` curly bracket!

File: danpitrowiski/hw1-madlibs/js/madlib-console.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	$( "#listTitle" ).empty().prepend("<h3>Favorites Startup Names</h3>");
+	$( "#favorites" ).empty().css('display','block');
+	$( "#xForY" ).css('display','none');
+	$( "#save" ).css('display','none');
+	$( "#print" ).css('display','none');
+
+	// Show all names in the favorites array
+	for (i=(favorite.length-1);i>0;i--){
+		$( "#favorites" ).append( "<p>"+favorite[i]+"</p>" );
+	}
```
Comment (Borgaard) [inline danpitrowiski/hw1-madlibs/js/madlib-console.js:73]: y'know what's real cool about `let`? It creates a block scope, so yer `i` never gets loose (not that it does in your code here, but it's a pretty neat bonus to ES6).

**Comment by Borgaard** [comment]
YA YA OKAY, ya beeped and ya booped, and ya bonus'd all the bonuses. I have just _ONE_ thing to say to you:
Those pumpkin crisps you brought were rather delicious. 


(_it's all very good code, like 5/5 stars; nicely done - especially with madlibs and bubbles of beer_ 🍻 )

File: danpitrowiski/hw1-99-bottles-of-beer/app.js (hunk 2 above)
Comment (DanPitrowiski) [reply danpitrowiski/hw1-99-bottles-of-beer/app.js:22]: ooo is the `else` suppose to directly follow the `if` is that why it's all separated here?

File: danpitrowiski/hw1-99-bottles-of-beer/css/style.css (hunk 1 above)
Comment (DanPitrowiski) [reply danpitrowiski/hw1-99-bottles-of-beer/css/style.css:141]: I should have included my inspirational codepen in my code too

File: danpitrowiski/hw1-madlibs/js/madlib-console.js (hunk 3 above)
Comment (DanPitrowiski) [reply danpitrowiski/hw1-madlibs/js/madlib-console.js:73]: Are there places here where I should have used `let` instead are `var`?

**Pull request merged**